license_manager/apps/subscriptions/event_utils.py

```diff
@@ -176,6 +180,12 @@ def track_event(lms_user_id, event_name, properties):
             "Event {} for user_id {} not tracked because SEGMENT_KEY not set".format(event_name, lms_user_id)
         )
 
+    print(f"hasattr? {hasattr(settings, 'KAFKA_ENABLED')}")
+    print(f"enabled? {settings.KAFKA_ENABLED}")
+    if hasattr(settings, "KAFKA_ENABLED") and settings.KAFKA_ENABLED:
```

Comment:
Can you use `getattr `here with a default?
```suggestion
    if getattr(settings, "KAFKA_ENABLED", False):
```